generator-configs/okx/openapi/go/rest.yaml

```diff
@@ -0,0 +1,16 @@
+gitUserId: openxapi
+gitRepoId: okx-go/rest
+inputSpec: ./specs/okx/openapi/rest.yaml
+templateDir: ./templates/okx/openapi/go
+additionalProperties:
+  packageName: rest
+  packageVersion: 0.1.0
+  withGoMod: false
+files:
+  signing.mustache:
+    destinationFilename: signing.go
+openapiGeneratorIgnoreList:
+  - .travis.yml
+  - git_push.sh
+nameMappings:
```

Comment:
this is for binance, if okx API doesn't have properties named `M` and `m` in a same object, you don't need it.